server/checkdoc.js

```diff
@@ -0,0 +1,31 @@
+const Doc = require('doc');
+
+const inputdoc = (req, res) => {
+  let docinst = new Doc({
+    name: req.body.name,
+    file: req.body.text,
+    owner_id: req.body.owner_id,
+    locked: false,
+    revision_id: 0,
+    last_revised: null
+  });
+  let tabooSchema = new Schema({
+    word: String
+  })
+  taboo = mongoose.model('Taboo', tabooSchema);
+  try {
+    const results = await taboo.find({});
```

Comment:
Do not use await while the rest of the code uses call backs

Reply:
changed to using callback